app/shared/services/contentful-content.service.ts

```diff
@@ -92,6 +103,27 @@ export class ContenfulContent {
       .commit();
   }
 
+  private getLatestItems<T>(request: ContentfulRequest, limit: number
```

Comment:
`latestItems` are sorted by `sys.createdAt` by default, but if we'd need some custom sorting there is an `order` method for sorting records.